qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AllergyPage.java

```diff
@@ -23,6 +23,8 @@
 	private static final By ALLERGY_STATUS = By.cssSelector(".allergyStatus");
 	private static final By ADD_NO_KNOWN = By.xpath("//*[contains(text(),'No Known Allergy')]");
 	private static final By DELETE_NO_KNOWN = By.className("delete-action");
+	private static final By RETURN_BUTTON = By.cssSelector("#content > button.cancel");
```

Comment:
is the element retrievable without `>` within the cssSelector?

Reply:
> is the element retrievable without `>` within the cssSelector?

The test fails with this error 
` Tests in error: 
  Allergies management(Allergies Management): Expected condition failed: waiting for visibility of element located by By.cssSelector: #content button.cancel (tried for 120 second(s) with 500 milliseconds interval)(..)
 `

Reply:
So basically if I remove that angle bracket, the element will not be visible.